tools/wptrunner/wptrunner/testdriver-extra.js

```diff
@@ -81,8 +97,15 @@
         for (let actionSequence of actions) {
             if (actionSequence.type == "pointer") {
                 for (let action of actionSequence.actions) {
-                    if (action.type == "pointerMove" && action.origin instanceof Element) {
-                        action.origin = {selector: get_selector(action.origin)};
+                    if (action.type == "pointerMove" && typeof(action.origin) != 'string') {
```

Comment:
Could you add a comment here to indicate what string type for origin refers to. It was "pointer" and "viewport" vs element object. Right?

Reply:
Yes, I will add a comment